src/texdocdialog.ui

```diff
@@ -6,32 +6,72 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>363</width>
-    <height>174</height>
+    <width>527</width>
+    <height>484</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Packages Help (Texdoc)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
-   <item row="1" column="0" colspan="3">
-    <widget class="QTableWidget" name="tbPackages">
-     <property name="columnCount">
-      <number>2</number>
+   <item row="1" column="0" colspan="2">
+    <widget class="QSplitter" name="splitter_2">
+     <property name="orientation">
+      <enum>Qt::Horizontal</enum>
      </property>
-     <column>
-      <property name="text">
-       <string>Package</string>
-      </property>
-     </column>
-     <column>
-      <property name="text">
-       <string>Caption</string>
+     <widget class="QTableWidget" name="tbPackages">
+      <attribute name="horizontalHeaderStretchLastSection">
+       <bool>true</bool>
+      </attribute>
+      <column>
+       <property name="text">
+        <string>Packages</string>
+       </property>
+      </column>
+     </widget>
+     <widget class="QSplitter" name="splitter">
+      <property name="orientation">
+       <enum>Qt::Vertical</enum>
       </property>
-     </column>
+      <widget class="QTextBrowser" name="PackageInfoBrowser"/>
+      <widget class="QWidget" name="">
+       <layout class="QVBoxLayout" name="verticalLayout">
+        <property name="spacing">
+         <number>0</number>
+        </property>
+        <item>
+         <layout class="QHBoxLayout" name="LanguagesLayout">

```

Comment:
the ususal naming strategy is something like lyLanguages or similar.
lower case first letter helps readability.